src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerTextView.cs

```diff
@@ -42,7 +42,8 @@ internal partial class DebuggerTextView : IWpfTextView, IDebuggerTextView, IText
             // The editor requires the current top buffer.
             // TODO it seems to be a hack. It should be removed.
             // Here is an issue to track: https://github.com/dotnet/roslyn/issues/31189
-            _innerTextView.Properties.AddProperty(CompletionRoot, bufferGraph.TopBuffer);
+            // Starting debugging for the second time, we already have the property set.
```

Comment:
I'd rephrase this comment: "we choose to overwrite anything else that's there, in case if this property already exists. It shouldn't, but it might.".